test/empty_dump.sql

```diff
@@ -284,7 +284,7 @@ COPY public.roles_users (user_id, role_id) FROM stdin;
 --
 
 COPY public.users (id, name, email, butt_user, butt_pw, password, active) FROM stdin;
-1	testuser	testuser@lahmacun.hu	buttuser	buttpw	testpassword	t
+1	testuser	testuser@lahmacun.hu	buttuser	buttpw	$2b$12$bz/hCRWRMjNeHGr66wYZO.a/HHhenAHZaLECKmVYPFLSVTzvC/l9a	t
```

Comment:
@tuz666 sure it's not a plain text public password which grants the srongest access rights? I'm note I expected it to go somehow via GH actions secrets, but you're the expert.

Reply:
This empty db is used only for the CI and that is 'testpassword' after hashing, it was needed in order to get the API token.
If you give me access to set up GH Actions secrets then I will try to store these stuff there :)

Reply:
Oh, I see, you're right, it's not sensitive then (cause this user/pwsd is not in our public arcsi instances).